src/main/java/ch/njol/skript/expressions/ExprPlayerlistHeaderFooter.java

```diff
@@ -44,17 +44,17 @@
 @Description("The message above and below the player list in the tab menu.")
 @Examples({"set all players tab list header to \"Welcome to the Server!\"",
 			"send \"%the player's tab list header%\" to player",
-			"reset all player's tab list header"})
+			"reset all players's tab list header"})
 @Since("INSERT VERSION")
-@RequiredPlugins("Spigot 1.13 or newer")
+@RequiredPlugins("1.13 or newer")
```

Comment:
Consider Minecraft 1.13 or newer. Just to avoid confusing anyone.

Reply:
^ but that isn't the case for other elements